src/03-modality-agnostic-files.md

```diff
@@ -264,15 +264,16 @@ imaging acquisition sequence (each run `.nii[.gz]` file) within one session.
 Each `.nii[.gz]` file should be described by at most one row. Relative paths to
 files should be used under a compulsory `filename` header. If acquisition time
 is included it should be under `acq_time` header. Datetime should be expressed
-in the following format `2009-06-15T13:45:30` (year, month, day, hour (24h),
-minute, second; this is equivalent to the RFC3339 "date-time" format, time zone
-is always assumed as local time). For anonymization purposes all dates within
-one subject should be shifted by a randomly chosen (but common across all runs
-etc.) number of days. This way relative timing would be preserved, but chances
-of identifying a person based on the date and time of their scan would be
-decreased. Dates that are shifted for anonymization purposes should be set to a
-year 1925 or earlier to clearly distinguish them from unmodified data. Shifting
-dates is RECOMMENDED, but not required.
+in the following format `2009-06-15T13:45:30[.000000]` (year, month, day,
```

Comment:
Can we here or somewhere state that the number of fractional digits is up to the user?

Reply:
What do you think of "No specific precision is required for fractional seconds, but the precision should be consistent across the dataset."

Reply:
> No specific precision is required for fractional seconds, but the precision should be consistent across the dataset

:+1:  with a capital SHOULD

Reply:
Done.